setup.cfg

```diff
@@ -30,11 +29,10 @@ patch_emoji = [fix], [patch]
 branch = main
 
 [options]  # generated by wipac:cicd_setup_builder: python_requires, packages
-python_requires = >=3.11, <3.12
+python_requires = >=3.12, <3.13
 install_requires =
 	jsonschema
 	motor
-	oms-mqclient[rabbitmq]  # needed for examples/request_task.py
```

Comment:
Maybe make an `[examples]` install option?

Reply:
it's a good idea. Unfortunately, the `oms-mqclient` is behind on python releases (doesn't support py 3.12). I believe that's constrained for a pulsar library. I think that conditional install options are possible in `pyproject.toml`

Reply:
Then I guess make an issue and come back to it?

Reply:
https://github.com/Observation-Management-Service/ewms-workflow-management-service/issues/20